steps/hearing/the-hearing/TheHearing.js

```diff
@@ -34,9 +34,9 @@ class TheHearing extends SaveToDraftStore {
 
     return [
       answer(this, {
-        question: this.content.cya.attendHearing.question,
+        question: this.content.cya.attendHearing[`question${this.suffix}`],
         section: sections.theHearing,
-        answer: titleise(content.cya.attendHearing[this.fields.attendHearing.value])
+        answer: titleise(content.cya.attendHearing[this.fields.attendHearing.value + this.suffix])
```

Comment:
might as well use the backtick for this as well, for consistency

Reply:
@benouaer Tested and pushed. Didn't work when I first tried initially